docs/_posts/2020-02-07-boltwash.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Bolt + Wash"
+description: "Navigate your Bolt inventory in Wash"
+author: michaelsmith
+---
+
+If you're not using Puppet Bolt, [check it out](https://puppetlabs.github.io/bolt/).
+
+If you're using Bolt, then we created the [Boltwash](https://github.com/puppetlabs/boltwash) plugin that lets you navigate your inventory. Let's check it out
+```
+wash . > stree bolt
+bolt
+└── [group]
+    └── [target]
+        └── fs
+```
+
+## Exploring Boltwash
+
+As you can see, the Bolt plugin organizes Bolt targets as groups. It also mounts something called `fs` (a view of the filesystem on that target). Let's explore it a bit more.
+```
+wash . > cd bolt
+wash bolt > ls
+all/
+my-docker-app/
+my-linux-nodes/
+my-windows-nodes/
+```
+
+I have an inventory setup organizing some machines by where they run. Your groups would be organized however you like. The `all` group will appear for everyone, it contains all targets in your inventory (it's an implicit group that can be used in Bolt). Let's explore a particular target.
+```
+wash bolt > tree -L 2 my-linux-nodes
+my-linux-nodes
+├── node1
+│   └── fs
+└── node2
+    └── fs
+wash bolt > cd my-linux-nodes/node1/fs
+wash bolt/my-linux-nodes/node1/fs > ls
+bin/
+boot/
+dev/
+etc/
+home/
+...
+```
+
+Great, I can see the files on that system, and I didn't have to do any additional setup. I can go look at config or watch logs
+```
+wash bolt/my-linux-nodes/node1/fs > cat etc/hosts
+127.0.0.1 localhost
+...
+wash bolt/my-linux-nodes/node1/fs > tail -f var/log/syslog
```

Comment:
Think an example with `find` would be useful? Maybe if logs generated by a plan had a certain regex?

Reply:
Maybe, but kind of out of juice for the day.

Reply:
I don't have any obvious ideas for that that seem better than the grep example.

Reply:
No worries, not super important.